xdsl/transforms/experimental/ConvertStencilToLLMLIR.py

```diff
@@ -303,6 +304,14 @@ def match_and_rewrite(self, op: ExternalLoadOp, rewriter: PatternRewriter,
         pass
 
 
+class TrivialExternalStoreOpCleanup(RewritePattern):
+
+    @op_type_rewrite_pattern
+    def match_and_rewrite(self, op: ExternalStoreOp, rewriter: PatternRewriter,
+                          /):
+            rewriter.replace_matched_op([], [])
```

Comment:
Why not erase the op? :sweat_smile:

Reply:
what's the API call for this?

Reply:
`rewriter.erase_matched_op()` :slightly_smiling_face:

Reply:
Thanks, done :)